try-it.md

```diff
@@ -33,11 +33,24 @@ permalink: /try-it.html
 
 ### 1.1. Prerequisites
 
-- System with CentOS 8 Stream or Ubuntu 20.04
+- System with CentOS 9 Stream or Ubuntu 22.04
 - Bare metal preferred, as we will be creating VMs to emulate bare metal hosts
 - Run as a user with passwordless sudo access
 - Minimum resource requirements for the host machine: 4C CPUs, 16 GB RAM memory
 
+For execution with VMs
+
+- Setup passwordless sudo access
+```
+  $ sudo visudo
+```
+- Include this line at the end of the sudoers file 
+```console
+  username  ALL=(ALL) NOPASSWD: ALL
+```
+- Save and exit
+- Manually **enable nested virtualization** if you don't have it enabled in your VM
+
```

Comment:
Let's also include the command and output to check if its enabled or not.

Reply:
That would be great, but the resources i found don't point to a particular command that checks if it is enabled or not?
Can you please help with this